include/deal.II/grid/manifold.h

```diff
@@ -273,6 +273,15 @@ class Manifold : public Subscriptor
   /// @{
 
   /**
+   * TODO:
+   */
+  virtual
+  Point<spacedim>
+  get_new_point(  const Point<spacedim> &P1,
+                  const Point<spacedim> &P2,
+                  double alpha) const;
```

Comment:
Is this function simply a shortcut for the other one? I guess it makes sense to have this given that we want to interpret the general new-point operation in terms of pairwise reductions.

For consistency, please rename the arguments using lower-case `p1`, `p2`.